src/modules/ROOT/pages/changelog/202303/02.json

```
@@ -0,0 +1,277 @@
+{
+  "meta": {
+    "releasedOn": "2023.06.08"
+  },
+  "added": [
+    {
+      "what": [
+        "option for transparent color in the color picker used in {APP_NAME_ADMIN}, and {APP_NAME_REPORTER}."
+      ],
+      "why": [
+        "to give users more configuration options when selecting color or background color for an element."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1211"
+      ]
+    },
+    {
+      "what": [
+        "the option to set a record list filter as a preset filter for a user.",
+        "The filter can be applied using the 'Filters' dropdown menu."
+      ],
+      "why": [
+        "to give the user the ability to pick a pre-configured filter."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1227/commits/aa28be29cba69efce55028fa08c3a6c9a45ccc3d"
+      ]
+    },
+    {
+      "what": [
+        "menu with 'more' options (delete, un-delete, and permissions) to resource lists in {APP_NAME_COMPOSE}, {APP_NAME_ADMIN}, {APP_NAME_REPORTER}, and {APP_NAME_WORKFLOW}."
+      ],
+      "why": [
+        "so the user doesn't have to scroll horizontally to access all resource options."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/46f5455bd0ae67b9de2159f9d04b054a711832e1"
+      ]
+    },
+    {
+      "what": [
+        "a stale data error in case resources like a workflow are updated whilst a different user is editing the old version of the same workflow."
+      ],
+      "why": [
+        "to help reduce configuration errors due to multiple people editing the same resource at the same time."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/2529cf337c991cc13a2a2e15f2e84b5a8abab4a0"
+      ]
+    },
+    {
+      "what": [
+        "button to select all records in a record list and preform the actions edit, delete, un-delete as well as executing workflows."
+      ],
+      "why": [
+        "to provide the user with an easier way to edit, delete, un-delete, and execute workflows on multiple records."
```

```suggestion
        "to provide the user with an easier way to edit, delete, un-delete, and execute workflows on records across multiple pages."
```

This reflects it better imo; we already had record selections that only really worked over a single/few pages; this now handles all of them